src/auth/models/users-model.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = (sequelize, DataTypes) => {
+  sequelize.define('User', {
+    username: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    password: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+  });
+};
```

Comment:
If you use a block body with an arrow function, you need a return statement. Alternatively, get rid of the outside curly brackets and put the `sequelize.define` on the same line as your arrow.